lein-modules

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# A script that walks the tree of Leiningen projects and does $* on them
+# original: https://github.com/juxt/yada/blob/master/treelein
+
```

Comment:
Should you maybe `set -e` so that if any of the lein commands (or cds) fails, the script would stop?

Reply:
fixed